server/printing/SCEPrinter

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+#!/usr/bin/env python3
+# import printing_server.py
+import curses
+import argparse
+parser = argparse.ArgumentParser(description='Print the mock for 2D printer.')
+parser.add_argument('--mock-output', help="Displays Mock Output", )
+args = parser.parse_args()
+
+
+def print_visual(screen):
+    screen = curses.initscr()
+    height, width = screen.getmaxyx()
+    screen.border(0)
+
+    title = "2D PRINTER MOCK"
+    tx = width//2 - len(title)//2
+    ty = height-(height-2)
+    screen.addstr(ty, tx,
+                  title, curses.A_BOLD)
+
+    page = "Total Pages Printed: "
+    status = "Status:"
+    subwindow = screen.derwin(
+        20, 50, height-(height-5), (width//2)-50)
+    subwindow.box()
+    subwindow.immedok(True)
+    subwindow.refresh()
+    subwindow.addstr(2, 2, "Printer #1")
+    subwindow.addstr(4, 2, page)
+    subwindow.addstr(5, 2, status)
+
+    subwindow2 = screen.derwin(
+        20, 50, height-(height-5), width//2)
+    subwindow2.box()
+    subwindow2.immedok(True)
+    subwindow2.refresh()
+    subwindow2.addstr(2, 2, "Printer #2")
+    subwindow2.addstr(4, 2, page)
+    subwindow2.addstr(5, 2, status)
```

Comment:
might also be helpful to rename `subwindow` and `subwindow2` to something more helpful like `left_window` and `right_window`